mixer/pkg/cache/cache_test.go

```diff
@@ -54,20 +56,27 @@ func testCacheBasic(c Cache, t *testing.T) {
 		{Get, "Y", "34", true, Stats{Misses: 2, Writes: 3, Hits: 5}},
 
 		// ensure removing X works and doesn't affect Y
-		{Remove, "X", "", false, Stats{Misses: 2, Writes: 4, Hits: 5}},
```

Comment:
what caused these values to change (less by one number)? Remove is no longer a write ?

Reply:
Correct, removal and eviction are no longer being counted as writes.